js/renderingPicture.js

```diff
@@ -0,0 +1,37 @@
+import {createPhotos} from './mockPhotos.js';
+
+const dataCreatePicture = createPhotos();
+const SAMPLE = document.querySelector('#picture').content.cloneNode(true);
+const PICTURES = document.querySelector('.pictures');
+const fragment = document.createDocumentFragment();
+
+const createPictures = (data, sample) =>{
+  const sampleClone = sample.cloneNode(true);
+  const randomPhoto = sampleClone.querySelector('.picture__img');
+  randomPhoto.srс = data.url;
+  randomPhoto.alt = data.description;
+  const likes = sampleClone.querySelector('.picture__likes');
+  likes.textContent = data.likes;
+  const comments = sampleClone.querySelector('.picture__comments');
+  comments.textContent = data.comments.length;
+  fragment.appendChild(sampleClone);
+  PICTURES.appendChild(fragment);
+};
+for(let i = 0; i < dataCreatePicture.length ; i++){
```

Comment:
Как и было сказано выше, не нужно, чтобы модуль при первом же выполнении что-то рисовал или делал что-то кроме инициализации (под инициализацией в данном случае можно понимать вот определение всех констант и функций)

Т.е. мы должны завернуть нашу отрисовку тоже в функцию, назавем её `renderPhotos` (Можешь назвать `renderPictures` по аналогии, главное **сделай везде одинаково во всех модулях**)
```suggestion
// Функция у нас принимает массив данных фотографий
const renderPhotos = (photos) => {
 // создается пустой фрагмент
  const fragment = document.createDocumentFragment();

  for (let i = 0; i < photos.length; i++) {
// для каждой фотографии создается элемент-клон и этот элемент добавляется в наш пустой фрагмент
    fragment.appendChild(createPicture(photos[i]));
  }
  // под конец наш заполненный фотографиями-элементами фрагмент один раз добавляется в наш html
  picturesContainer.appendChild(fragment);
}
```

Reply:
Ну и попробуй не писать `for` и использовать `forEach` например так:
```suggestion
const renderPhotos = (photos) => {
  const fragment = document.createDocumentFragment();

  photos.forEach((photo) => fragment.appendChild(createPicture(photo)));

  picturesContainer.appendChild(fragment);
};
```

Reply:
+